torch_np/tests/numpy_tests/core/test_scalarmath.py

```diff
@@ -352,18 +357,10 @@ def test_float_modulus_corner_cases(self):
                     div, mod = op(fone, fzer)
                     assert_(np.isinf(div)) and assert_(np.isnan(mod))
 
-    def test_inplace_floordiv_handling(self):
```

Comment:
why removed?

Reply:
This test is AFAU specific to combining signed and unsigned 64-bit integers

```
        # this only applies to in-place floordiv //=, because the output type
        # promotes to float which does not fit
        a = np.array([1, 2], np.int64)
        b = np.array([1, 2], np.uint64)
        with pytest.raises(TypeError,
                match=r"Cannot cast ufunc 'floor_divide' output from"):
            a //= b
```

Chainging the dtype of b does not cause any trouble:

```
In [13]: a = np.array([1, 2], np.int64) ; a //= b.astype('uint8'); a
Out[13]: array([1, 1])

In [14]: a = np.array([1, 2], np.int64) ; a //= b.astype('int32'); a
Out[14]: array([1, 1])

In [15]: a = np.array([1, 2], np.int64) ; a //= b.astype('int64'); a
Out[15]: array([1, 1])

```

Reply:
Right. As discussed, moving forward, it'd be good to put together all these test removals into one PR that can be merged with little review.